active-directory-b2c-wpf/MainWindow.xaml.cs

```diff
@@ -37,10 +37,10 @@ private async void SignInButton_Click(object sender, RoutedEventArgs e)
                     .WithAccount(GetAccountByPolicy(accounts, App.PolicySignUpSignIn))
                     .ExecuteAsync();
 
-                DisplayBasicTokenInfo(authResult);
+                DisplayUserInfo(authResult);
                 UpdateSignInState(true);
             }
-            catch (MsalServiceException ex)
+            catch (MsalException ex)
```

Comment:
We probavly need to document is somewhere in the wiki/the doc as this is odd compared to the rest.